requirements.txt

```diff
@@ -1,2 +1,10 @@
-django
-Pillow
+Django==2.2.1
+entrypoints==0.3
+flake8==3.7.7
+isort==4.3.20
+mccabe==0.6.1
+Pillow==6.0.0
+pycodestyle==2.5.0
+pyflakes==2.1.1
+pytz==2019.1
+sqlparse==0.3.0
```

Comment:
Does it use`entrypoints` `mccabe` `pycodestyle` `pyflakes` `sqlparse` and where does it use ?